hadoop-hdfs-project/hadoop-hdfs-nfs/src/test/java/org/apache/hadoop/hdfs/nfs/nfs3/TestViewfsWithNfs3.java

```diff
@@ -316,6 +316,10 @@ public void testNfsRenameSingleNN() throws Exception {
         nn1.getRpcServer().getFileInfo("/user1/renameSingleNN");
     Assert.assertEquals(statusBeforeRename.isDirectory(), false);
 
+    Path successFilePath = new Path("/user1/renameSingleNNSucess");
+    if (hdfs1.exists(successFilePath)) {
+      hdfs1.delete(successFilePath, false);
+    }
```

Comment:
You don't need to check if the file exists. Just calling delete is sufficient:
https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/filesystem/filesystem.html#boolean_delete.28Path_p.2C_boolean_recursive.29

Reply:
Thanks @aajisaka for pointing that out! Updated the fix.